mlflow/gateway/schemas/chat.py

```diff
@@ -29,35 +22,14 @@ class BaseRequestPayload(ChatTools, RequestModel):
 }
 
 
-class RequestPayload(BaseRequestPayload):
-    messages: list[RequestMessage] = Field(..., min_items=1)
-
+class RequestPayload(ChatCompletionRequest, RequestModel):
     class Config:
         if IS_PYDANTIC_V2_OR_NEWER:
             json_schema_extra = _REQUEST_PAYLOAD_EXTRA_SCHEMA
         else:
             schema_extra = _REQUEST_PAYLOAD_EXTRA_SCHEMA
 
 
-class ResponseMessage(ResponseModel):
-    role: str
-    content: Optional[ContentType] = None
-    tool_calls: Optional[list[ToolCall]] = None
-    refusal: Optional[str] = None
-
-
-class Choice(ResponseModel):
-    index: int
-    message: ResponseMessage
-    finish_reason: Optional[str] = None
-
-
-class ChatUsage(ResponseModel):
```

Comment:
note: Moving these under `mlflow.types.chat` so LangChain flavor does not rely on gateway module.